### DISCUSSION TITLE
Where did Nic go?

### DISCUSSION THREAD

**Discussion by Wasupmacuz** [nicolai-wachenschwan/arc-overhang-prusaslicer-integration#86] [General]
Hi everyone,

I've noticed that it's been nearly two years since the last code update in this repository. The community has been actively addressing known issues and submitting pull requests, but it seems uncertain whether they will be reviewed and merged.

I've personally contributed several improvements, including performance optimizations and bug fixes, which I believe could significantly benefit the project. However, the lack of feedback or acknowledgment is making it challenging to sustain my motivation to continue contributing. As a result, my activity here may appear inconsistent, although I've continued refining the script in my own environment.

I'm not sure if Nicolai is still actively involved with the project or if there are other reasons for the current inactivity. To ensure the project remains vibrant and useful, I propose that we explore community-driven solutions. This could involve trying to reach out to Nicolai to understand his current availability or considering alternatives like forking the repository to continue development under community management.

I'm hopeful that together, we can find a way to keep this project alive and thriving.

Thank you for considering this proposal.

**Comment by jperry004** [comment]
Yeah we should fork it or ask him to take over. Or best to start over from scratch. But I don't totally understand all the functions.

**Reply by Wasupmacuz** [reply]
I think a rework is necessary. At the very least, comments should be abundant so it's obvious what any code does.
